### DISCUSSION TITLE
Provide better documentation

### DISCUSSION THREAD

**Discussion by stsp** [dosemu2/dosemu2#1642] [General]
Hi @rouyng
Not sure what have happened,
but our documentation discussion
somehow disappeared.
So I am restarting it.
I have no explanation, have never
seen tickets or discussions
disappearing w/o any traces.

**Comment by rouyng** [comment]
Odd, perhaps the discussion creator was sick of receiving notifications and decided to delete the discussion instead of disabling notifications? 

Regardless, I plan to do more work converting README.html to markdown and will bump this discussion when I make some progress.

**Reply by stsp** [reply]
Arrgh, its quite entirely
outdated...
Well, hope with it being
online we'll find more ppl
willing to update it.
At least definitely no one
have done so in 10 years
of it being in an offline
format, so we need to try
your approach anyway. :)

**Reply by stsp** [reply]
But at least you can ask
me if the particular chapter
is worth translating at all.
Many should be removed,
so it would be better to save
some time of yours.
